src/PHPSemVerChecker/Analyzer/ClassMethodAnalyzer.php

```diff
@@ -109,6 +112,29 @@ public function analyze(Stmt $contextBefore, Stmt $contextAfter)
 					);
 				}
 
+				if ($methodBefore->returnType !== $methodAfter->returnType) {
+					$class = ClassMethodCaseChanged::class;
+					if ($methodBefore->returnType === null) {
+						$class = ClassMethodReturnTypeAdded::class;
+					} elseif ($methodAfter->returnType === null) {
+						$class = ClassMethodReturnTypeRemoved::class;
+					} elseif ($methodAfter->returnType->name !== $methodBefore->returnType->name) {
```

Comment:
```suggestion
					} elseif (strcasecmp($methodAfter->returnType->name, $methodBefore->returnType->name) !== 0) {
```
In PHP classes name are case insensitive so while the casing might be different, it shouldn't result in a `ClassMethodReturnTypeChanged`.